iota/commands/extended/__init__.py

```diff
@@ -22,3 +22,4 @@
 from .replay_bundle import *
 from .send_transfer import *
 from .send_trytes import *
+from .is_reattachable import *
```

Comment:
I'd like to keep these guys in alphabetical order.  Makes it easier to tell at a glance if something is missing.

Reply:
done